go.mod

```diff
@@ -50,7 +50,7 @@ require (
 	github.com/naokibtn/go-ses v0.0.0-20150122091825-74908b78cc76
 	github.com/onsi/ginkgo v1.10.2 // indirect
 	github.com/sirupsen/logrus v1.4.2 // indirect
-	github.com/soh335/go-test-redisserver v0.0.0-20170912004345-f716f0901533
+	github.com/soh335/go-test-redisserver v0.1.0
```

Comment:
This old soh335/go-test-redisserver refers to garyburd/redigo, so I updated this too.